Package.swift

```diff
@@ -79,7 +79,8 @@ let package = Package(
     ],
     products: [
         .library(name: "Crypto", targets: ["Crypto"]),
-        .library(name: "_CryptoExtras", targets: ["_CryptoExtras"]),
+        .library(name: "_CryptoExtras", targets: ["CryptoExtras"]),
```

Comment:
One for discussion - the README is set up that this library **won't** exist but I've left it here for discussion if we want to keep it to maintain backwards compatibility or not or release this with the upcoming major version

Reply:
I think retaining the backward compatibility is valuable: while the underscore is present, we've tried to avoid gratuitously breaking things and it'd be nice to do so here.

Reply:
Ok I've updated the README. In terms of getting this to work with CMake do I need an shim for `_CryptoExtras` that points to `CryptoExtras`?